src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/MSBuildProject.cs

```diff
@@ -87,9 +100,17 @@ ProjectComponents GetSDKProjectComponents()
                         components |= ProjectComponents.Web;
                     }
 
-                    if (Sdk.Equals(MSBuildConstants.DesktopSdk, StringComparison.OrdinalIgnoreCase) ||
-                        GetPropertyValue("UseWPF").Equals("true", StringComparison.OrdinalIgnoreCase) ||
-                        GetPropertyValue("UseWindowsForms").Equals("true", StringComparison.OrdinalIgnoreCase))
+                    if (GetPropertyValue("UseWPF").Equals("true", StringComparison.OrdinalIgnoreCase))
+                    {
+                        components |= ProjectComponents.Wpf;
+                    }
+
+                    if (GetPropertyValue("UseWindowsForms").Equals("true", StringComparison.OrdinalIgnoreCase))
+                    {
+                        components |= ProjectComponents.WinForms;
```

Comment:
With new .NET CLIs, Windows desktop apps don't have to use the WindowsDesktop SDK. But they still need `<UseWpf>` or `<UseWindowsForms>`, so we need those properties to also `components |= ProjectComponents.WindowsDesktop`.

Reply:
Thanks for clarifying Mike